### ISSUE TITLE
JS Legacy mode warning when using the latest version

### ISSUE THREAD

**Issue by sureshg** [JetBrains/markdown#121]
As per this issue https://github.com/JetBrains/markdown/issues/75 , the library supports `JS-IR` targets. But using the latest version (`0.4.1`) of this library and `Kotlin 1.9.0` throws the following warning for JS(IR) target compilation. 

```
> Task :api-client:transformCommonMainDependenciesMetadata
w: The dependency org.jetbrains:markdown:0.4.1 was published in the legacy mode. Support for such dependencies will be removed in the future. See https://kotl.in/0b5kn8 for details.
```
Any suggestions to fix this warning or is this an issue with the library itself?

**Comment by valich** [comment]
AFAIU this is a deprecation warning issued when using a library built with old kotlin versions. We need to bump used kotlin dependency. It's severely old now.

@FirstTimeInForever let's upgrade kotlin version to at least 1.8.20, or better 1.9.0?

**Comment by FirstTimeInForever** [comment]
@sureshg I am preparing the release with the update to Kotlin `1.9.0`. Currently, I've only published the snapshot version, so we can test that everything is OK. Please try the `0.4.2-SNAPSHOT`.

**Comment by sureshg** [comment]
@FirstTimeInForever thanks very much.. is there any doc on how to configure Gradle to test the snapshot versions? I couldn't find it in maven central snapshot repo.

**Comment by FirstTimeInForever** [comment]
@sureshg Adding `maven("https://oss.sonatype.org/content/repositories/snapshots/")` to your `repositories {}` should work.

**Comment by sureshg** [comment]
@FirstTimeInForever thanks..seems like it fixed it. I am not seeing the warning message now.

**Comment by sureshg** [comment]
`0.5.1-SNAPSHOT` is also working fine.

**Comment by FirstTimeInForever** [comment]
I've released stable `0.5.0` that should fix all of the issues. Consider using it instead of the `0.5.1-SNAPSHOT`.
